tests/functional/test_basic.py

```diff
@@ -545,3 +545,22 @@ def test_hook_collect_submodules(pyi_builder, script_dir):
 # Test that PyInstaller can handle a script with an arbitrary extension.
 def test_arbitrary_ext(pyi_builder):
     pyi_builder.test_script('pyi_arbitrary_ext.foo')
+    
+def test_option_runtime_tmpdir(pyi_builder):
```

Comment:
Side-note: "Issue #2221" is not the reason, why this test should be skipped. The reason is something like "option --runtime-tmpdir not implemented for win32".

(Putting only the ticket numbers into the reason is a bad idea anyway, since it makes it harder to realize when the reason is not longer valid. I know, there are some reasons referring to ticket numbers only, but I ~~assume~~hope these are bug-tickets, not pull requests.)